rosetta-source/src/main/resources/cdm-sample-files/functions/business-event/quantity-change/full-termination-equity-swap-func-output.json

```diff
@@ -946,35 +905,6 @@
                   }
                 }
               },
-              "extraordinaryEvents" : {
```

Comment:
Are these diffs expected?

Reply:
yes. The FpML mapping to extraordinaryEvent has been intentionally deactivated as an interim state. It will be rerouted in a second phase to the legalAgreement components.